### ISSUE TITLE
CI failure email not sufficiently actionable

### ISSUE THREAD

**Issue by mhofman** [Agoric/Agoric#4662]
## What is the Problem Being Solved?

Currently the email sent when a CI job fails on master (introduced by https://github.com/Agoric/agoric-sdk/pull/4613), are pretty terse, and don't contain enough information for them to be considered relevant by team members. This issue is to explore including additional information.

Refs: #4459

## Description of the Design
TBD

## Security Considerations
None

## Test Plan
N/A

**Comment by mhofman** [comment]
@FUDCo @Chris-Hibbert, you both voiced that you'd like these emails to be more actionable. Do you have ideas of what would make them so?

**Comment by mhofman** [comment]
One suggestion was a link to the PR for which the master commit was created. One possibility to extract this may be to grab it from the commit subject (which should be formatted as `subject (#1234)`).

Including the commit message would be worth considering as well. The commit author probably not since that'll likely be the mergebot account.

**Comment by Chris-Hibbert** [comment]
I find the current message inscrutable. I don't know where to start in analyzing any of the messages I see. I don't know how to tell whether it's even plausible that I could be the culprit. I assume that's true for the other recipients.

I hope this doesn't mean that more than one engineer will spend time on each message figuring out whether there is a problem, and who should address it. I'd be happier having on-call rotations of engineers assigned to triage and assign these notices than having multiple people work in parallel to decide whether it's theirs, even if most of them decide that they don't need to spend more time. 

I don't know what causes the problem, or what context github has from which to decide. I agree that it's bad to leave things broken, but this diffuses responsibility rather than assigning or localizing it.

**Comment by mhofman** [comment]
Unfortunately it's pretty difficult to assign responsibility. Maybe a little sentence explaining that if this is a new occurence, the problem likely originated by the most recent PR.

That coupled with linking to the PR / commit which triggered this email might go a long way. I doubt we can easily figure out if previous commits similarly failed. If we could we maybe could include those results to bring some sense of if this is a new issue or not.

I also believe some of this confusion may be alleviated with more experience, and maybe a pinned status in the keybase coordination channel.

**Comment by FUDCo** [comment]
I think @Chris-Hibbert expressed my own concerns rather well.  CI failure generally shouldn't the fault of some commit.  If it was, then the testing prior to merge should have caught the problem and kicked it back to the PR author before getting to the whatever stage is sending these emails.  So any of these CI failures should be attributed to problems with the CI system itself.  On the other hand, if we are in a broken state, then it may be that rolling back or tweaking a commit will unbreak it, and the person whose commit it was that tripped over the problem may be in the best position to do that.  But I'm leaning towards thinking that part of the issue is that we don't have clear ownership of the CI process itself.

**Comment by mhofman** [comment]
> the testing prior to merge should have caught the problem and kicked it back to the PR author before getting to the whatever stage is sending these emails

Unfortunately some CI jobs are just too expensive to run for every update in a PR, and we haven't found a way to configure mergify to run some jobs only before landing, and keep the PR open if it doesn't land.

**Comment by arirubinstein** [comment]
With mergify and the new CI changes, maybe this email won't trigger as much?

**Comment by mhofman** [comment]
Unfortunately we're not there yet, and we've had to back out the mergify queue changes. The Keybase/Slack bot has a little more actionable info, but the situation is not good yet.

**Comment by Tartuffo** [comment]
We are now sending via Slack.
